utils/RudderContext.js

```diff
@@ -34,7 +34,13 @@ class RudderContext {
       screen.width = window.screen.width;
       screen.height = window.screen.height;
       screen.density = window.devicePixelRatio;
-      this.userAgent = navigator.userAgent;
+      // detect brave browser and append to the user agent
+      if (navigator.brave && navigator.brave.isBrave.name === "isBrave") {
+        const version = navigator.userAgent.match(/(Chrome)\/([\w\.]+)/i)[2];
+        this.userAgent = `${navigator.userAgent} Brave/${version}`;
```

Comment:
The browser name should be "brave" instead of "Brave" to match Amplitude's regex pattern

Reply:
"/ i" is there in the regex. So, it will be Case-insensitive. "Brave" or "brave" doesn’t  matters. Chrome and other browser follow the same way Chrome/version that's why I put it like this.

Reply:
My bad, I didn't realize '/i' at the end